rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -128,6 +132,16 @@ class PublisherBase
   size_t
   get_intra_process_subscription_count() const;
 
+  /// Manually assert that this Publisher is alive (for RMW_QOS_POLICY_MANUAL_BY_TOPIC)
```

Comment:
nitpick: punctuation

Reply:
I see this a few times elsewhere too, but I won't comment on each, please skim over the docblocks at some point and make sure complete sentences have punctuation.